packages/cli/src/databases/migrations/postgresdb/1690000000030-RemoveResetPasswordColumns.ts

```diff
@@ -0,0 +1,29 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+import { TableColumn } from 'typeorm';
+
+export class RemoveResetPasswordColumns1690000000030 implements ReversibleMigration {
```

Comment:
considering that all three migrations are identical (because you used the queryRunner helper functions 🎉 ), We should move this file to `packages/cli/src/databases/migrations/common/1690000000030-RemoveResetPasswordColumns.ts`, and import it from there into the DB specific `index.ts` files.

This is what I started doing in https://github.com/n8n-io/n8n/pull/6691